orchagent/orch.cpp

```diff
@@ -260,7 +260,18 @@ bool Orch::parseReference(type_map &type_maps, string &ref_in, string &type_name
         tokens = tokenize(ref_content, config_db_key_delimiter);
         if (tokens.size() != 2)
         {
-            SWSS_LOG_ERROR("malformed reference:%s. Must contain 2 tokens\n", ref_content.c_str());
+            if (tokens.size() == 1)
+            {
+                auto type_it = type_maps.find(tokens[0]);
+                if (type_it != type_maps.end())
+                {
+                    type_name = tokens[0];
+                    SWSS_LOG_INFO("Incomplete reference received: type_name:%s, object_name: missing\n", tokens[0].c_str());
```

Comment:
no need to add '\n' in the log. for the rest of the code not in this pull request that has '\n', it could be removed later.